RTN-008.tex

```diff
@@ -62,114 +63,113 @@
+Accordingly, bespoke custom processing will be required in order to generate timely and useful data products.
+Followup observations in the first 24 hours are among the most critical for identifying EM counterparts and constraining progenitor scenarios. 
 
-User access to raw and processed images is not required prior to 24 hours after the data are taken (\texttt{L1PublicT}; \texttt{LSR-REQ-0104} \citedsp{LSE-29}).  
-Moreover, \texttt{LSR-REQ-0126} specifies that images and catalog products except for alerts are embargoed for a \textit{minimum} of \texttt{L1PublicTmin} = 6 hours before release to anyone \citedsp{LSE-29}.
-Followup observations in the first 24 hours are among the most critical for constraining progenitor scenarios, so delays of this duration are unlikely to be viewed as acceptable by the community, particularly if they are not driven by technical limitations.
+Raw and processed LSST images are initially embargoed in both commissioning and operations.
+This precludes simply releasing the TOO images for community processing.
+Instead, timely custom processing of TOO data must be performed by Rubin staff (\S \ref{sec:processing}).
 
-Current experience with DECam in particular suggests there will be a major community demand for user-generated processing, even starting with the raw imaging. 
-Given the relatively few exposures expected, the compute and/or data-transfer needs for users are not onerous.
-If the LSST imaging data is available in a timely manner, users may elect to perform processing beyond standard LSST processing, including differencing against templates from other surveys (e.g., DECaLS), cross-band image differencing to identify sources of extreme color, differencing of nightly stacks, alternative image differencing algorithms, etc.
+We assume that any triggers would be on relatively well-localized GW sources (a few LSST pointings), and so this additional data processing is well within operational compute margins.
 
-Timely access to the images themselves is necessary to enable this user processing.
-Given the high profile and priority of the science, we suggest that the LSST Operations Project attempt to secure an agency exemption from \texttt{LSR-REQ-0126} for TOO data specifically to enable rapid release ($<$1 hour) of the imaging data.
-This may require developing alternative data release interfaces.
+\section{Recommendations}
 
-The question of data rights is also present here; we assume that by default access to imaging data would continue to be restricted to LSST Users \citedsp{LDO-13}.
+%If the LSST imaging data is available in a timely manner, users may elect to perform processing beyond standard LSST processing, including differencing against templates from other surveys (e.g., DECaLS), cross-band image differencing to identify sources of extreme color, differencing of nightly stacks, alternative image differencing algorithms, etc.
```

Comment:
Even the best case would be 80hr. But you have this commented out.

Reply:
Right, these were leftovers from the earlier version which (pre OGA) argued that we should consider making the images themselves widely available on timescales short enough for rapid followup.